### DISCUSSION TITLE
Sveltekit/Supabase error

### DISCUSSION THREAD

**Discussion by biscotty666** [sveltejs/kit#10489] [Q&A]
I'm following the guide at https://supabase.com/docs/guides/getting-started/tutorials/with-sveltekit. After installing auth-ui-svetle and auth-ui-shared I get the following error message:

```
Internal server error: Failed to resolve entry for package "@supabase/auth-ui-svelte". The package may have incorrect main/module/exports specified in its package.json: No known conditions for "." specifier in "@supabase/auth-ui-svelte" package
```
Googling found this https://github.com/nextauthjs/next-auth/issues/6161 which seems similar but suggests the situation is resolved.

I'd appreciate some help with this. Thanks. Brian

**Comment by regrettablemouse136** [comment]
@biscotty666 Did you solve this?

**Reply by biscotty666** [reply]
No, I gave up and forgot about it frankly since I got no response and there are so many alternatives.

**Comment by david-plugge** [comment]
This isn't an issue sveltekit can fix. Consider open an issue in the supabase auth ui repo instead
